packages/stats/gbstats/gbstats.py

```diff
@@ -211,20 +216,40 @@ def get_configured_test(
                 ),
             )
     else:
+        assert isinstance(
+            stat_a, type(stat_b)
+        ), "stat_a and stat_b must be of same type."
```

Comment:
Sorry to nit, I should have pointed this out initially, but this will pass even so long as stat_a is the same class *or a subclass* of stat_b. e.g is stat_a is a SampleMeanStatistic and stat_b is a Statistic, this will pass.

So I think we should instead just do `assert type(stat_a) is type(stat_b)`

Reply:
accepted